### ISSUE TITLE
415 Unsupported media type only through Fuel

### ISSUE THREAD

**Issue by rehovicova** [kittinunf/fuel#781]
Hi, I have a server I am trying to reach to. If I open a browser and simply enter the url of my endpoint, I can see the response. If I use curl `curl localhost:9090/api/some-get-endpoint` I can see the response.Even when I try it in Postman, and create a simple GET request, it works. But if I try to reach the server using Fuel I get `415 Unsupported Media Type`.

This is the code I am running:
```
    FuelManager.instance.basePath = "http://$host:$port/api"

    "/position"
                .httpGet()
                .responseString(handler).join()
```

I am sure I am missing somehing stupid, but I don't see what...

**Comment by SleeplessByte** [comment]
The API server thinks there is a content-type given, which is... weird because there is no body. Do you have access to the API server?

**Comment by rehovicova** [comment]
I mean... The weird part is that every other client I used can connect to it, except Fuel. Which to me sounds like Fuel is sending something different than all the other clients do. Maybe you can copy the server code and try if it works for you?

**Comment by SleeplessByte** [comment]
I agree that this is weird. I don't know (haven't been in this codebase for a while) if we add something, but it really shouldn't add a content-type and/or content-length to get requests. Maybe it does, but that's a bug.

**Comment by rehovicova** [comment]
I am kinda in a time pressure here, so I guess I will swap from Fuel to something else. See it that will solve it. Otherwise if you have any other suggestion, it would be appreciated.

**Comment by SleeplessByte** [comment]
I personally won't be able to debug this right now, unfortunately. It's possible that this is Fuel; I would switch it out if that helps you right now. You can always come back to this :)

**Comment by rehovicova** [comment]
I swapped to kohttp, the problem is gone. If you want to further investigate the issue with Fuel I was using version `2.2.3`and then I swapped to the latest `2.3.0`. Both of them were buggy for me.

**Comment by kittinunf** [comment]
I will try to fork the C++ and see what is going on there. Sorry! and Thanks for the report! 🙇
